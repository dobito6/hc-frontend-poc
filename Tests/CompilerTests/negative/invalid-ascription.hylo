//! no-std

public fun main() {
  var a: Void
  var b: a
}
