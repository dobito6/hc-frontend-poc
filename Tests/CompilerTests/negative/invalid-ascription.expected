negative/invalid-ascription.hylo:5.10-11: error: expression does not denote a type
  var b: a
         ^
